.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['20.x', '22.x', '23.x']
```

Comment:
I don't think we should remove 20.x just yet, since eslint-config/plugin is used for our github-actions, which I believe is still required to be Node 20.